tools/pipelines/test-perf-benchmarks.yml

```diff
@@ -70,160 +85,311 @@ stages:
+            targetType: 'inline'
+            workingDirectory: $(toolAbsolutePath)
+            script: |
+              cp ${{ variables.testWorkspace }}/.npmrc . ;
+              npm install @ff-internal/aria-logger;
+              npm i;
+              npm run build:compile;
 
-        - task: Npm@1
-          displayName: 'Consolidate output files - execution time'
+        # Download Test Files
+        # Download Artifact - Test Files
+        - task: DownloadPipelineArtifact@2
+          displayName: Download test files
           inputs:
-            command: 'custom'
-            workingDir: ${{ variables.buildDirectory }}
-            customCommand: 'run test:benchmark:copyresults'
+            source: specific
+            project: internal
+            pipeline: ${{ variables.artifactPipeline }}
+            preferTriggeringPipeline: true
+            allowPartiallySucceededBuilds: true
+            runVersion: latestFromBranch
+            runBranch: $(Build.SourceBranch)
+            artifact: test-files
+            path: $(Pipeline.Workspace)/test-files/
+
+        - ${{ each execTestPackage in parameters.executionTestPackages }}:
+
+          # Download package artifacts
+          - task: DownloadPipelineArtifact@2
+            displayName: Download test package
+            inputs:
+              source: specific
+              project: internal
+              pipeline: ${{ variables.artifactPipeline }}
+              preferTriggeringPipeline: true
+              allowPartiallySucceededBuilds: true
+              runVersion: latestFromBranch
+              runBranch: $(Build.SourceBranch)
+              artifact: pack
+              patterns: "**/${{ replace(replace(execTestPackage, '@', '' ), '/', '-') }}-*.tgz"
+              path: $(Pipeline.Workspace)/client/pack
+
+          # Install test package
+          - task: Bash@3
+            displayName: Install Test Package
+            inputs:
+              targetType: 'inline'
+              workingDirectory: ${{ variables.testWorkspace }}
+              script: |
+                echo "Install ${{ execTestPackage }}"
+                TEST_PACKAGE_FILE_PATTERN=${{ replace(replace(execTestPackage, '@', '' ), '/', '-') }}-*.tgz
+                TEST_PACKAGE_PATH_PATTERN=$(Pipeline.Workspace)/client/pack/scoped/$TEST_PACKAGE_FILE_PATTERN
+
+                echo $TEST_PACKAGE_FILE_PATTERN
+                echo $TEST_PACKAGE_PATH_PATTERN
+
+                echo `ls -1 $TEST_PACKAGE_PATH_PATTERN | wc -l`
+                echo `ls $TEST_PACKAGE_PATH_PATTERN`
+
+                # if [[ `ls -1 $TEST_PACKAGE_PATH_PATTERN | wc -l` -eq 1 ]]; then
+                TEST_PACKAGE_TGZ=`ls $TEST_PACKAGE_PATH_PATTERN`
+                # else
+                #   ls -1 $TEST_PACKAGE_PATH_PATTERN
+                #   echo "##vso[task.logissue type=error]Test package '${{ execTestPackage }}' not found, or there are more then one found"
+                # fi
```

Comment:
This should be uncommented, right?

Reply:
I meant to come back to this problem. This was the problem: to download the `packages/experimental/tree` package , we use the blob: `fluid-experimental-tree-*.tgz`. However, there are two packages that match that blob:
`fluid-experimental-tree-graphql-0.0.0-108119-test.tgz` and `fluid-experimental-tree-0.0.0-108119-test.tgz`. So when we finally come down to this check, there exist two paths that cause the `-eq 1` check to fail. 

I'm changing the pattern matching blob to include the semver pattern to avoid this problem. So, we will now match and download paths containing: `fluid-experimental-tree-?.?.?-*.tgz`.

Reply:
The glob pattern will not work for version segments with more than one digit, but I think that's ok for now. Historically we don't have any patch versions going into two digits (ignoring the time when we were on `0.59.#000`), minor ones will rarely/never get there, and major ones are very far from it. So I'm good with this for now.